src/features/chain-connection/WalletConnection.tsx

```diff
@@ -38,12 +40,10 @@ const WalletConnection = () => {
         <span
           className={twJoin(
             isConnected ? "font-medium" : "font-bold px-1",
-            "whitespace-nowrap",
+            "whitespace-nowrap normal-case",
```

Comment:
I feel like this would be an appropriate variant to put into Button.tsx, but it's totally fine here

Reply:
yeah - could have been a variant, but it seems to be an exception (all other buttons are uppercase, regardless of variant etc.), so I felt this was possibly the better call?